### PR TITLE
automake: add automake/1.16.2

### PR DESCRIPTION
**Pull request by madebr**
Specify library name and version:  **automake/1.16.2**

- [x] I've read the [guidelines](https://github.com/conan-io/conan-center-index/wiki#how-to-submit-a-pull-request) for contributing.
- [x] I've followed the [PEP8](https://www.python.org/dev/peps/pep-0008/) style guides for Python code in the recipes.
- [x] I've used the [latest](https://github.com/conan-io/conan/releases/latest) Conan client version.
- [x] I've tried at least one configuration locally with the
      [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

**Comment by uilianries** [comment]
> For my own knowledge, what's the difference between `tools.os_info.is_windows` and `self.settings.os == "Windows"`
> 
> LGTM

tools.os_info.is_windows checks your current OS (e.g. os.system() == "Windows"), settings.os is about the profile, you can lie it.

File: recipes/automake/all/conanfile.py [hunk 1]
```diff
@@ -90,34 +90,34 @@ def package_info(self):
         self.output.info("Appending PATH env var with : {}".format(bin_path))
         self.env_info.PATH.append(bin_path)
 
-        bin_ext = ".exe" if self.settings.os_build == "Windows" else ""
+        bin_ext = ".exe" if self.settings.os == "Windows" else ""
 
         aclocal = os.path.join(self.package_folder, "bin", "aclocal" + bin_ext)
-        if self.settings.os_build == "Windows":
+        if tools.os_info.is_windows:
```
Comment (danimtb) [inline recipes/automake/all/conanfile.py:96]: `tools.unix_path()` will have no effect if the OS is different from windows in Conan 1.25. Maybe we can wait a bit a remove this checks

Comment (madebr) [reply recipes/automake/all/conanfile.py]: I waited 24 hours. Is that enough?

Comment (danimtb) [reply recipes/automake/all/conanfile.py]: there is a minor 1.25.1 waiting, so... but the issue with `unix_paths()` only affect in special case so I guess we can move forward this one

**Comment by madebr** [comment]
@SSE4 @danimtb 
This is ready for merging.
If you merge this before #807, I can bump the requirement of libtool.

**Pull request merged**